account_invoice_change_currency/views/account_invoice.xml

```diff
@@ -12,6 +12,10 @@
                 <button aria-label="Opened" type="object" name="action_account_change_currency" class="btn-link" states="draft" groups="account.group_account_manager">
                     <i title="Update Currency: Will convert the amount lines to the new currency set" class="fa fa-fw fa-refresh"/>
                 </button>
+                <div>
+                    <label for="custom_rate"/>
+                    <field name="custom_rate" class="oe_inline"/>
```

Comment:
No label and use 3 items together

`Currency: [ currency_id  ] [rate] ( )`

`currency_id with label`, `rate without labe`, `sync button`